src/ClientFlags/ClientFlags.cpp

```diff
@@ -54,11 +54,14 @@ ABSL_FLAG(bool, enforce_full_redraw, false,
           "Enforce full redraw every frame (used for performance measurements)");
 
 // VSI
-ABSL_FLAG(std::string, connection_target, "",
-          "Instance and process in the form <pid>@<instance>, where <instance> is either an "
-          "instance ID or an instance display name. Specify this to skip the connection setup and "
-          "open the main window instead. If either the instance or the process ID can't be found "
-          "or deployment is aborted by the user Orbit will exit with return code -1 immediately.");
+ABSL_FLAG(std::string, target_process, "",
+          "Process name or path. Specify this together with --target_instance to skip the "
+          "connection setup and open the main window instead. If the process can't be found or "
+          "deployment is aborted by the user Orbit will exit with return code -1 immediately.");
```

Comment:
I think we should probably specify what the "process name" is because that's a very overloaded term on Linux. In our case we consider the process name the so-called "command name" from the /proc/pid/comm file. But that's a weird thing because it's max 16 characters long. If it's not stated I would assume this also matches `/proc/pid/cmdline` because most tools support both. (For example `top` and `ps` both have options to toggle between `comm` and `cmdline`)

So I see three options:
1. Make it clear in the description that it matches `comm`
2. Also consider `cmdline`
3. Not support either of those. I assume you would only need the matching against the process path and matching against the name is only there for potential other users?

Reply:
One more thing: We should probably mention that the first matching process is taken and we don't scan through process with any particular order.

Reply:
Regarding the first comment: For VSI, we will receive the process path, and as you suspected I've added support for the process name to make it easier for users to use this as well. I'm matching against what we also show as "Name" in the connection window. I'm not sure if we need to be more specific here TBH, this should allow you to specify whatever you see as "Name" when you select a process in Orbit.

Regarding order: Good point, but I wanted to create a follow-up PR that would go through the processes in descending PID order so we make sure to select the most recent process if there's multiple matches. I would then update the description.

Reply:
I think its fine to not mention the comm & cmdline details in this explanation text. I guess we use the same as the default in top & ps and then I am fine with calling it process name (If we dont use the default, then we should consider changing that). I would also be okay to not support process name, but just path. Maybe the Orbit documentation is a good place to put the details about Orbit using comm.

> descending PID order so we make sure to select the most recent process

Is the highest PID always the more recent process?

Reply:
Yes, it's my understanding that they are guaranteed to be incremental.

Reply:
They are usually incremental but can wrap around. Ideally we would use the start time for sorting. But I don't know how much work that would be.

Reply:
Right, I didn't consider that. But I would just ignore this case - it requires a very unlikely combination of circumstances for this to happen (PID needs to wrap around & two processes with the same name need to be running). For our primary use case (starting from VSI), this shouldn't happen at all since VSI will take care of managing the process.